### PR TITLE
Implemented One City Challenge

### PR DESCRIPTION
**Pull request by BenBooth1344**
Implemented One City Challenge

### CODE REVIEW

**Comment by BenBooth1344** [comment]
No, only players are limited to one city.
I think I forgot to prevent capturing of settlers. I'll fix it when I get back to my computer.

**Comment by BenBooth1344** [comment]
Fixed so OCC can no longer capture settlers.
Should be ready to go!

**Comment by Smashfanful** [comment]
Nice. Don't forget, though: you wrote Destory instead of Destroy

File: core/src/com/unciv/logic/battle/Battle.kt [hunk 1]
```diff
@@ -254,9 +254,11 @@ class Battle(val gameInfo:GameInfo) {
...
 
     private fun captureCivilianUnit(attacker: ICombatant, defender: ICombatant){
-        // barbarians don't capture civilians, City-states don't capture settlers
-        if(attacker.getCivInfo().isBarbarian()
-                || (attacker.getCivInfo().isCityState() && defender.getName()==Constants.settler)){
+        // barbarians, City-states, OCC don't capture civilians don't capture settlers
+        var cannotCaptureSettler = (attacker.getCivInfo().isBarbarian()
+                || attacker.getCivInfo().isCityState()
+                || attacker.getCivInfo().isPlayerOneCityChallenger())
+        if(cannotCaptureSettler && defender.getName()==Constants.settler){
```
Comment (yairm210) [inline core/src/com/unciv/logic/battle/Battle.kt]: This means that barbarians will be able to capture other civilians, not what we want...

**Comment by yairm210** [comment]
Also you have conflicts, please resolve :)

**Comment by BenBooth1344** [comment]
Will do.

**Comment by BenBooth1344** [comment]
Should be ready to go

**Pull request merged**